### PR TITLE
Create a new "Dedicated Load Balancer Firmware update" article.

### PR DESCRIPTION
**Pull request by maxim-clc**

### CODE REVIEW

File: Network/dedicated-load-balancer-firmware-upgrade.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+To address existing bugs and bring to light set of new features periodically a new firmware for your virtual load balancer is released. To make those improvements available you would need to go through the upgrade  procedure. In case of HA pair upgrade no down time would be required, however the proper planning is the key. The high level preview of the upgrade process looks as follows - disconnect Secondary node from synchronization, upgrade it, convert into Primary node; disconnect original Primary node from synchronization, upgrade it, turn on synchronization, convert into Primary node again.
+
+### Prerequisites
+
+* you have HA dedicated load balancers which have been configured under your account
+* you have ability to login into appliance with administrative credentials
+
+### Preparation
+
+First of all, you would need to create a ticket and ask NOC staff to upload the version of the firmware you are interested in upgrading to.
```
Comment (chrislittle) [inline Network/dedicated-load-balancer-firmware-upgrade.md]: assume that the reader isn't sure how to open a ticket with the NOC.  provide a link to how to submit a support request.  you may even want to include a code snippet to copy and paste for the email of the body like ecosystem does and we do (see https://www.ctl.io/knowledge-base/network/centurylink-cloud-guide-to-openvpn-access-server/#reserving-client-vpn-dhcp-pool-in-the-control-portal for example)

File: Network/dedicated-load-balancer-firmware-upgrade.md [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+  "date": "12-11-2015",
+  "author": "Maxim Volkov",
+  "attachments": [],
+  "related-products" : [],
+  "contentIsHTML": false,
+  "sticky": false
+}}}
+
+### Introduction
+
```
Comment (chrislittle) [inline Network/dedicated-load-balancer-firmware-upgrade.md]: overall comment, should there be a section for troubleshooting (at a minimum how they get help if something goes wrong) and how to confirm the upgrade is complete?  

lastly, have you validated the NOC wants customers upgrading their own firmware as the default option?  my experience getting multiple upgrades is they did it on request (I didn't have to).  i'd think a customer would prefer that model so if its an option we should present the customer the details on how to get the upgrade done by our teams.  this kb is still good to show but we should make sure we at least let them know there is another choice.

**Comment by maxim-clc** [comment]
Suggested improvements been introduced.

**Comment by maxim-clc** [comment]
Pushed suggested improvements.

**Pull request merged**